www/js/controller/home.controller.js

```diff
@@ -1,10 +1,33 @@
 angular.module('vamosJuntas').controller('HomeController',
- ['$scope', 'placeFactory', 'addressFactory',  function($scope, placeFactory, addressFactory) {
-  placeFactory.fetchPlaces().then(function(response) {
-    $scope.places = response.data;
-  });
+ ['$scope', 'placeFactory', 'addressFactory', '$cordovaGeolocation', '$ionicLoading', function($scope, placeFactory, addressFactory, $cordovaGeolocation, $ionicLoading) {
+
+
   $scope.search = {};
   $scope.addresses;
+  $scope.errorMessage = false;
+
+  $ionicLoading.show();
+
+  var coordinates = {};
+  var posOptions = {timeout: 5000, enableHighAccuracy: false};
+
+  $cordovaGeolocation
+    .getCurrentPosition(posOptions)
+    .then(function (position) {
```

Comment:
can we move the $codovaGeolocation.getCurrentPosition() to a factory? usually the controller should not know the business logic.